server/index.php

```diff
@@ -50,6 +50,10 @@
 	exit;
 });
 
+if (!defined('DEBUG')) {
+	define('DEBUG', null);
```

Comment:
Shouldn't it be `false` by default ?

Reply:
Firstly, to be clear it was like this before my change here.

But I think `null` is intentional. The example config file in the README has this:

```const DEBUG = __DIR__ . '/debug.log';```

This tells me that `DEBUG` isn't a boolean but rather a file path. And it would make sense that if you don't want debug, `null` would be way to indicate that.

Reply:
Your explanation make sence for me. 

Still if DEBUG is a file path, it's default value might be `""` instead of `null`. What do you think about that ?